Kitodo/src/main/java/de/sub/goobi/forms/ProzessverwaltungForm.java

```diff
@@ -1986,12 +1986,12 @@ public void setShowClosedProcesses(boolean showClosedProcesses) {
         this.showClosedProcesses = showClosedProcesses;
     }
 
-    public void setShowArchivedProjects(boolean showArchivedProjects) {
-        this.showArchivedProjects = showArchivedProjects;
+    public void setShowInactiveProjects(boolean showInactiveProjects) {
+        this.showInactiveProjects = showInactiveProjects;
     }
 
-    public boolean isShowArchivedProjects() {
-        return this.showArchivedProjects;
+    public boolean isShowInactiveProjects() {
```

Comment:
As you changing the method name and signature, please add JavaDoc to this public method.